packages/venia-concept/src/reducers/mockAccount.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
Please move this and all other mocks/fixtures into a __tests__ subfolder like __fixtures__. Add to the Jest exclude pattern if you have to.